docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+---
+version: '3.6'
+services:
+  bot:
+    build:
+      context: .
+      dockerfile: Dockerfile.bot
+    command: bash -c "go build -o bin/bot && bin/bot -t ${BOT_TOKEN}"
+    # without values will pull the value from your host ENV of the same name
+    environment:
+      - STOCK_API_URL
+      - COIN_API_URL
+      - ERR_INVALID_CMD
```

Comment:
the fuck are these

Reply:
env vars ya idiot

Reply:
no bitch im asking what are the `err_invalid_cmd` for? are you checking for some response code from the APIs? 

why can't we hardcode these in because it wouldn't different from environment to environment

Reply:
it's the error message for an invalid command, as a reminder for you

`lol retarded?`